lib/onfido/resource.rb

```diff
@@ -39,7 +39,8 @@ def make_request(options)
 
       response = RestClient::Request.execute(request_options)
 
-      parse(response)
+      #response should be parsed only when there is a response expected
+      parse(response) if response.code != 204 #no_content
```

Comment:
In that case we will miss if the response is json and for any reason it is returned as empty body and an error should be raiseed then